invenio_app_rdm/theme/assets/semantic-ui/js/invenio_app_rdm/landing_page/ShareOptions/ShareModal.js

```diff
@@ -31,7 +31,7 @@ export class ShareModal extends Component {
       usersResults: undefined,
       groupsResults: undefined,
       linksResults: undefined,
-      activeTab: 0,
+      activeTabKey: "",
```

Comment:
are these relevant changes regarding login required?

Reply:
No, changes in this file are the bug fix of "Settings" tab reported by Lars

Reply:
Can you please split them in a separate commit just for clarity please? Basically, keep in one commit only the removal of the `login_required`...

Reply:
Done!